storage_service/locations/models/package.py

```diff
@@ -994,6 +996,9 @@ def finish_reingest(self, origin_location, origin_path, reingest_location, reing
         )
         reingest_full_path = os.path.join(ss_internal.full_path, reingest_path)
 
+        # Take note of whether new version of AIP should be compressed
+        to_be_compressed = os.path.isfile(os.path.join(ss_internal.full_path, reingest_path))
```

Comment:
Nitpick: You can reuse reingest_full_path from the line above instead of doing the os.path.join